linker/linker_test.go

```diff
@@ -35,6 +39,20 @@ import (
 	"github.com/bufbuild/protocompile/reporter"
 )
 
+var (
+	protocPath = "../internal/testdata/protoc/22.0/bin/protoc"
```

Comment:
Since this is only used in one place (line 2265), let's inline this into a function closer to that spot. Something like so:
```go
func getProtocPath() {
    if runtime.GOOS == "windows" {
        return "protoc"
    }
    return "../internal/testdata/protoc/22.0/bin/protoc"
}
```